### ISSUE TITLE
The default skin is intrusive to the point MathML is impossible to display

### ISSUE THREAD

**Issue by mytskine** [tinymce/tinymce#5581]
When a plugin opens a dialog, the Oxyde skin (TinyMCE's default skin) applies a CSS to every element of the dialog (not by inheritance, but with the `*` selector). This breaks some displays that require external CSS, like displaying formulas with MathML.

See <http://fiddle.tinymce.com/Ufhaab> for an example.
The formula inside the TinyMCE dialog should look like the formula outside TinyMCE.

This behavior is caused by two rules (27 directives) in `skins/ui/oxide/skin.css`.
```css
.tox *:not(svg):not(rect) { /* */ }
.tox *:not(svg):not(rect) { /* */ }
```
Both apply their directive to every element at every depth under`.tox`, instead of relying on the usual inheritance. This is very heavy for the browser, and it is very hard to overrule.

**Comment by lostkeys** [comment]
The reason we have the reset rules is to prevent outside styles to leak into TinyMCE. Your fiddle is seems broken (it is showing nothing) so if you can fix that we can try to help you out.

**Comment by mytskine** [comment]
Fixed the demo: <http://fiddle.tinymce.com/Ufhaab/3>. Note that the formula outside the TinyMCE dialog is clean.

BTW, the tinymce-fiddle UI is a rather misleading, with no indicator of unsaved changes, and no confirmation on leave.

**Comment by lostkeys** [comment]
Yeah, sorry there is no good solution to your problem at the moment. As said, we need to reset all styles within TinyMCE to prevent outside styles leaking into and breaking TinyMCE. Thus TinyMCE does not support injecting external UI into the TinyMCE UI which this technically is. We want that to be possible though and we're looking into how, but I don't have timeframe for you at this time.

**Comment by prathamVaidya** [comment]
@mytskine have you found a solution to prevent the tox styles?


@lostkeys what about just adding a `.no-tox-style` class?

```css
.tox *:not(svg):not(rect):not(.no-tox-css) { /* */ }
.tox *:not(svg):not(rect):not(.no-tox-css) { /* */ }
```

This way at least elements that don't need the tox style can use the `.no-tox-style` class

**Comment by tiny-ci-servers** [comment]
This issue was closed because it has been stalled for 30 days with no activity.
